### PR TITLE
HL-203

### PR DESCRIPTION
**Pull request by bazmatic**
https://dltx.atlassian.net/browse/HL-203

- Faucet and test accounts funded on deployment
- Fixed logic error in Registry contract (could not add Vaults)

### CODE REVIEW

**Review by bitcoinbrisbane — DISMISSED** [review]

File: contracts/Registry.sol [hunk 1]
```diff
@@ -31,10 +31,7 @@ contract Registry {
 
     function addVault(address vault) external onlyTokenHolders {
         address assetAddress = IVault(vault).asset();
-        require(
-            IVault(_vaultByAsset[assetAddress]).asset() == address(0),
-            "addVault: Vault with this asset token already added"
-        );
+        require(_vaultByAsset[assetAddress] == address(0), "addVault: Vault with this asset token already added");
```
Comment (rellfy) [inline contracts/Registry.sol:34]: Maybe require `vault != address(0)` so that you can't push to the array by accident?

Comment (Preston-Harrison) [reply contracts/Registry.sol:34]: I think that would just throw when you tried to get the asset, so I don think it's necessary

File: hardhat.config.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ export default {
 					apiKey: process.env.ETHERSCAN_API_KEY
 				}
 			},
+			gasMultiplier: 1.1,
```
Comment (Preston-Harrison) [inline hardhat.config.ts:54]: why?

Comment (bazmatic) [reply hardhat.config.ts:54]: Because otherwise Goerli deploys tend to fail.

File: contracts/Registry.sol (hunk 1 above)
Comment (bazmatic) [reply contracts/Registry.sol:35]: I've added it anyway to be explicit

**Review by Preston-Harrison — APPROVED** [review]

**Review by rellfy — DISMISSED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**